R/mergecategoriescontainingranges.R

```diff
@@ -422,10 +462,10 @@ identifyClosedOrOpenBoundariesFromText <- function(label) {
     global.closed.phrases <- c("or equal")
     
     lower.boundary.open.phrases <- c("less than", "under", "fewer than", "lower than", "below", "<", "smaller than", "younger than")
-    lower.boundary.closed.phrases <- c( "up to", "<=", "and less", "and under", "and lower", "and below", "and smaller")
+    lower.boundary.closed.phrases <- c(paste0(c("and ", "or "), rep(c("less", "under", "lower", "below", "smaller", "younger", "fewer")), each = 2), c( "up to", "<="))
```

Comment:
This has a typo, suspect the `)` in the wrong spot. E.g. when executing the 2s appear.
```
ower.boundary.closed.phrases <- c(paste0(c("and ", "or "), rep(c("less", "under", "lower", "below", "smaller", "younger
", "fewer")), each = 2), c( "up to", "<="))
> lower.boundary.closed.phrases
[1] "and less2"    "or under2"    "and lower2"   "or below2"    "and smaller2"
[6] "or younger2"  "and fewer2"   "up to"        "<="
```

Reply:
I'd consider adding tests to check these phrases properly caught.
E.g. `expect_equal(identifyClosedOrOpenBoundariesFromText("less than"), "lower boundary ...")`

Reply:
Also unrelated to these commits but in the code later in this function, the `else if` blocks are redundant if the `return` statements are used within each previous block.
A final `return` on the last line of a function is redundant too, could simply be `"ambiguous"`